vue-frontend/src/components/contributions/DevOps.vue

```diff
@@ -0,0 +1,213 @@
+<template>
+  <section class="tw-my-[100px]">
+    <p
+      class="tw-container tw-text-center tw-text-[1.875rem] lg:tw-text-[3.438rem] tw-leading-[2.438rem] lg:tw-leading-[5.156rem] tw-font-inter-bold tw-text-black-core/[0.87]"
+    >
+      DevOps: CI/CD and Auto Deployment
+    </p>
+    <div class="tw-block lg:tw-hidden tw-mt-[3rem]">
+      <swiper :slidesPerView="1.325" :centeredSlides="true" :spaceBetween="0">
+        <swiper-slide v-for="(blog, index) in devOps" :key="index">
+          <div
+            class="tw-transition-all tw-duration-200 tw-ease-out tw-scale-[0.75]"
+          >
+            <img
+              @click="openBlog(blog)"
+              :src="blog.image[0]"
+              :srcset="`${blog.image[0]} 400w, ${blog.image[1]} 800w`"
+              class="tw-relative tw-w-full tw-h-full tw-object-cover tw-rounded-[10px]"
+              alt="favourite"
+            />
+            <div
+              class="time tw-absolute tw-top-0 tw-right-0 tw-rounded-bl-[5px] tw-bg-white tw-px-3 tw-py-1 tw-font-inter-medium tw-text-[0.875rem] tw-leading-[1.313rem]"
+            >
+              {{ blog.time }}
+            </div>
+          </div>
+        </swiper-slide>
+      </swiper>
+    </div>
+    <div
+      class="tw-container tw-hidden lg:tw-flex tw-flex-col tw-gap-10 tw-mt-16"
+    >
+      <div class="tw-flex tw-flex-row tw-gap-10">
+        <div v-for="blog in devOps.slice(0, 2)" :key="blog.id">
+          <div
+            class="tw-relative tw-w-full tw-h-full tw-group"
+            @click="openBlog(blog)"
+          >
+            <img
+              :src="blog.image[0]"
+              :srcset="`${blog.image[0]} 400w, ${blog.image[1]} 800w`"
+              alt="favourite"
+              class="tw-w-full tw-h-full tw-object-cover tw-rounded-[20px] tw-cursor-pointer"
+              loading="lazy"
+            />
+            <div
+              class="gradient-class tw-absolute tw-left-[0] tw-bottom-[0] tw-w-[55%] tw-h-[20%] tw-rounded-b-[10px] tw-opacity-1 tw-transition-all tw-duration-1000 tw-ease group-hover:tw-w-[100%] group-hover:tw-h-[85%]"
+            ></div>
+            <div
+              class="tw-absolute tw-bottom-10 tw-p-[1rem] tw-opacity-0 tw-text-white tw-font-inter-medium tw-transition-all tw-duration-1000 tw-translate-y-[50%] group-hover:tw-translate-y-[0%] group-hover:tw-transition-all group-hover:tw-duration-1000 group-hover:tw-opacity-[1]"
```

Comment:
Text and div animations timings are not aligned. They should come on same time.
In current scenario, text are visible earlier then div